aas_core_codegen/main.py

```diff
@@ -30,6 +31,19 @@ class Target(enum.Enum):
     TYPESCRIPT = "typescript"
     RDF_SHACL = "rdf_shacl"
     XSD = "xsd"
+    JSONLD_CONTEXT = "jsonld_context"
+    # NOTE (murloc6, 2023-10-06)
+    # JSON-LD (JSON Linked Data) Context Generation
```

Comment:
@Murloc6 (This is the comment I had in mind re comment above for the documentation. I'd move it into the docstring and remove it here. Sorry for the zigzag.)